DOCKER_NOTICE.md

```diff
@@ -0,0 +1,29 @@
+## Notice for Docker image
+
+This application provides container images for demonstration purposes.
+
+DockerHub: To be added once we have Docker publish workflow
+
+Eclipse Tractus-X product(s) installed within the image:
+
+Item-Relationship-Service
+
+- GitHub: https://github.com/eclipse-tractusx/item-relationship-service
```

Comment:
the reference to IRS is wrong

Reply:
Corrected: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/2/commits/486556999761b417d2a1ee05e7615c6a1243080a